hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -54,13 +56,10 @@ public void removeStagingTask(final Protos.TaskID taskID) {
   }
 
   public void removeRunningTask(Protos.TaskID taskId) {
-    if (!nameNode1TaskMap.isEmpty()
-        && nameNode1TaskMap.keySet().iterator().next().getTaskId().equals(taskId)) {
-      nameNode1TaskMap.clear();
-    } else if (!nameNode2TaskMap.isEmpty()
-        && nameNode2TaskMap.keySet().iterator().next().getTaskId().equals(taskId)) {
-      nameNode2TaskMap.clear();
-    }
+    log.info(String.format("Removing running task: %s", taskId));
+
```

Comment:
Why?  Stylistic preference?  Or is the micro-optimization of calls?  We'd save what a hasStatus() null check call or two if we do it your way.